ext/pdo/tests/pdo_035.phpt

```diff
@@ -0,0 +1,252 @@
+--TEST--
+PDO Common: PDORow + get_parent_class()
+--EXTENSIONS--
+pdo
+--SKIPIF--
+<?php
+$dir = getenv('REDIR_TEST_DIR');
+if (false == $dir) die('skip no driver');
+require_once $dir . 'pdo_test.inc';
+PDOTest::skip();
+?>
+--FILE--
+<?php
+if (getenv('REDIR_TEST_DIR') === false) putenv('REDIR_TEST_DIR='.__DIR__ . '/../../pdo/tests/');
+require_once getenv('REDIR_TEST_DIR') . 'pdo_test.inc';
+$db = PDOTest::factory();
+
+const TABLE_NAME = 'test_pdo_35_pdo_row';
```

Comment:
@Girgias 
There is also a way to do this only in the case of oci.

```suggestion
$db = PDOTest::factory();

if ($db->getAttribute(PDO::ATTR_DRIVER_NAME) === 'oci') {
    $db->setAttribute(PDO::ATTR_ORACLE_NULLS, PDO::NULL_TO_STRING);
}

const TABLE_NAME = 'test_pdo_35_pdo_row';
```

Reply:
However, if checking the behavior of `exists`, as in your test, `'0'` may be appropriate. Looks good as it is now.